r-network.tf

```diff
@@ -0,0 +1,29 @@
+resource "azurerm_virtual_network" "this" {
+  name                = "vnet-launchpad-prd-${local.location_short[var.location]}"
+  resource_group_name = var.resource_group_name
+  location            = var.location
+  tags                = var.tags
+
+  address_space = var.virtual_network_address_space
+  # dns_servers   = ["10.0.0.1", "10.0.0.1"]
```

Comment:
Remove unused commented code. This was in the skeleton because some clients need explicit `dns_servers`. In a module, this should be an input variable that defaults to an empty list.